src/reducers/existingThoughtChange.js

```diff
@@ -37,10 +34,23 @@ import {
   checkIfPathShareSubcontext,
   timeDifference
 } from '../util.js'
+import saveCurrentCursorOffset from '../util/saveCurrentCursorOffset.js'
```

Comment:
Notice how `util` functions are imported above. Rather than import this directly from `saveCurrentCursorOffset.js`, add it to `util.js` where all of the util functions are defined.